content/Arduino Cloud/Cloud Editor/If-you-see-an-error-when-performing-an-Over-the-Air-OTA-upload.md

```diff
@@ -0,0 +1,143 @@
+* [Board error - otastorageinitfail](#otastorageinitfail)
+* [Board error - serverconnecterrorfail](#serverconnecterrorfail)
+* [SHA256 mismatch - Unable to apply the new firmware. Previous firmware detected after restart](#sha256mismatch)
+* [SHA256 Unknown - Unknown Firmware detected after restart.](#sha256unknown)
+
+### Board error - errorwriteupdatefilefail {#errorwriteupdatefilefail}
+
+This error indicates that the device's memory partition table is improperly formatted.
+
+Follow these steps:
+
+1. Run the memory formatting sketch for your device architecture.
+1. Reinitialize the storage with this Sketch: [QSPIFormat.ino](https://github.com/arduino/ArduinoCore-mbed/blob/main/libraries/STM32H747_System/examples/QSPIFormat/QSPIFormat.ino)
+1. Reinstall the SSL certificates with this Sketch: [WiFiFirmwareUpdater](https://github.com/arduino/ArduinoCore-mbed/tree/main/libraries/STM32H747_System/examples/WiFiFirmwareUpdater)
+
+> [!TIP]
+> If you're using GIGA R1 WiFi, see [Configure GIGA R1 WiFi for Over-The-Air (OTA) uploads](https://support.arduino.cc/hc/en-us/articles/12370721200540-Configure-GIGA-R1-WiFi-for-Over-The-Air-OTA-uploads).
+
+### Board error - httpresponsefail {#httpresponsefail}
+
+This error can occur if the OTA update fails during the firmware download.
+
+Follow these steps:
+
+1. Ensure that the board has a stable connection to Arduino Cloud.
+1. Avoid extensive calls to the `delay()` function or other blocking code inside the `loop()` function of your sketch.
+1. Try deactivating the watchdog timer on the device.
+
+### Board error - otadownloadfail {#otadownloadfail}
+
+This error can occur if the OTA update fails during the firmware download.
+
+Follow these steps:
+
+1. Ensure that the board has a stable connection to Arduino Cloud.
+1. Avoid extensive calls to the `delay()` function or other blocking code inside the `loop()` function of your sketch.
+
+### Board error - otaheadercrcfail {#otaheadercrcfail}
+
+The header of the downloaded file does not match the expected one, likely indicating that the file was corrupted during the download.
+
+Follow these steps:
+
+1. Try to upload the sketch again.
+
+### Board error - otastorageinitfail {#otastorageinitfail}
+
+This error indicates that the device's memory partition table is improperly formatted.
+
+Follow these steps:
+
+1. Run the memory formatting sketch for your device architecture.
+1. Reinitialize the storage with this Sketch: [QSPIFormat.ino](https://github.com/arduino/ArduinoCore-mbed/blob/main/libraries/STM32H747_System/examples/QSPIFormat/QSPIFormat.ino)
+1. Reinstall the SSL certificates with this Sketch: [WiFiFirmwareUpdater](https://github.com/arduino/ArduinoCore-mbed/tree/main/libraries/STM32H747_System/examples/WiFiFirmwareUpdater)
+
+> [!TIP]
+> If you're using GIGA R1 WiFi, see [Configure GIGA R1 WiFi for Over-The-Air (OTA) uploads](https://support.arduino.cc/hc/en-us/articles/12370721200540-Configure-GIGA-R1-WiFi-for-Over-The-Air-OTA-uploads).
```

Comment:
Any changes resulting from the comments made [here](https://github.com/arduino/help-center-content/pull/393/files#diff-e08ad45a1eaaa5ef98f5ed20566bb3c4b8dd3b0534d7f3bc1222614b47380418R64-R75) should probably be applied to these lines as well.